### ISSUE TITLE
Bug when we load plan after save it in JLD2 format

### ISSUE THREAD

**Issue by mariusmjp** [bankofcanada/StateSpaceEcon.jl#15]
Hi,

I noticed an issue when I tried to save plan in JLD2 and reload it after (please see the following snapshot). 
![plan_save_and_load](https://user-images.githubusercontent.com/22647406/93507626-ecad3a00-f8eb-11ea-8eca-62594615c0a0.png)

**Comment by bbejanov** [comment]
Look at this discussion: https://discourse.julialang.org/t/a-future-for-jld2/42958/37 
The future of JLD2 is uncertain. Do you still think that using JLD2 is a good idea?

**Comment by mariusmjp** [comment]
Yes I also see the same concern before but the question is which format we should use ? Because it seems all pre-existing format have some limitation and it seems JLD2 is the most faster:  https://discourse.julialang.org/t/a-future-for-jld2/42958

**Comment by RoyCCWang** [comment]
Sorry for reviving this thread. Would [BSON.jl](https://github.com/JuliaIO/BSON.jl) work for your application? It also save closures/anonymous functions. Some idiosyncrasies:

- For nested arrays, BSON.jl saves it as an array of datatype `Any`'s. You'll need to do `convert()` yourself when loading from such a BSON file.

- If the anonymous function in a BSON file contains a package, then the user who loads the BSON file should have the same version of that package loaded in scope. Example: If a BSON file has a variable for `f = xx->MyPack.myfunc(xx)+3`, then when loading the file, the user must have done `import MyPack`, where`MyPack` must be the same version as the `MyPack` for `f` when `f` was saved.

**Comment by bbejanov** [comment]
I'm not sure that we'll ever support JLD2 or any other format for plans. As of f2d962022c69ddc3b1b9e9b4b5aacfff140fc4d4 we now have `importplan` and `exportplan`, which use a text format so that plans with the same model and range can also be compared visually with plain diff.
